tardis/simulation/base.py

```diff
@@ -421,25 +423,47 @@ def log_plasma_state(
         plasma_state_log["next_t_rad"] = next_t_rad
         plasma_state_log["w"] = w
         plasma_state_log["next_w"] = next_w
+        plasma_state_log.columns.name = "Shell No."
 
-        plasma_state_log.index.name = "Shell"
-
-        plasma_state_log = str(plasma_state_log[::log_sampling])
+        if self.check_notebook():
+            logger.info(f"\n\tPlasma stratification:")
+            logger.info(
+                display(
+                    plasma_state_log.iloc[::log_sampling].style.format("{:.3g}")
+                )
+            )
+        else:
+            output_df = ""
+            plasma_output = plasma_state_log.iloc[::log_sampling].to_string(
+                float_format=lambda x: "{:.3g}".format(x), justify="center",
+            )
+            for value in plasma_output.split("\n"):
+                output_df = output_df + "\t{}\n".format(value)
+            logger.info(f"\n\tPlasma stratification:\n\n{output_df}")
 
-        plasma_state_log = "".join(
-            ["\t%s\n" % item for item in plasma_state_log.split("\n")]
+        logger.info(
+            f"\n\tCurrent t_inner = {t_inner:.3f}\n\tExpected t_inner for next iteration = {next_t_inner:.3f}\n"
         )
 
-        logger.info("Plasma stratification:\n%s\n", plasma_state_log)
-        logger.info(f"t_inner {t_inner:.3f} -- next t_inner {next_t_inner:.3f}")
-
     def log_run_results(self, emitted_luminosity, absorbed_luminosity):
         logger.info(
-            f"Luminosity emitted = {emitted_luminosity:.5e} "
-            f"Luminosity absorbed = {absorbed_luminosity:.5e} "
-            f"Luminosity requested = {self.luminosity_requested:.5e}"
+            f"\n\tLuminosity emitted   = {emitted_luminosity:.3e}\n"
+            f"\tLuminosity absorbed  = {absorbed_luminosity:.3e}\n"
+            f"\tLuminosity requested = {self.luminosity_requested:.3e}\n"
         )
 
+    def check_notebook(self):
```

Comment:
Hello @andrewfullard, I would be creating a new PR for adding this function to the appropriate place 😄  
I will make this, such that it automatically takes import from that function 😉

Reply:
Right, but I want to see it working in this PR. Currently your screenshots are outdated.

Reply:
Sure, I will update the screenshots here 🚀

Reply:
Also, please make a docstring and test for this new function, and rebase your other PR where you relocate the function so that it follows the changes made here..

Reply:
I have updated the screenshots for the respective outputs.